testhelpers/unit_test_suite.go

```diff
@@ -17,6 +17,9 @@ func SetupTest() {
 	var err error
 
 	logrus.SetLevel(logrus.DebugLevel)
+	if os.Getenv("CI") == "true" {
+		logrus.SetLevel(logrus.ErrorLevel)
```

Comment:
Can you remind me what this is for? I don't recall anymore.

Reply:
The log output from the binlog events is seemingly causing CI to truncate the output. This just lowers the logging level unless needed for the tests